src/stepDefinitions/provisioningContext.js

```diff
@@ -306,8 +306,9 @@ Given('group {string} has been created', function (groupId) {
 })
 
 Given('these groups have been created:', function (dataTable) {
+  const groupIdsToCreate = dataTable.rows().slice(1)
```

Comment:
`curl -XPOST http://localhost:3000/execute -d '{"step":"Given these groups have been created:","table":[["groupname"],["HelloGroup"]]}' -H 'Content-Type: application/json'` creates two groups: `groupname` and `HelloGroup`, since `groupname` is only a table column header, it should be excluded.

Reply:
Rebase on top of https://github.com/owncloud/owncloud-test-middleware/pull/37. that will fix this issue